CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,90 @@
+# Changelog 1.4
+
+## 1.4.0 - 2021-11-03
+
+The NodePortLocal feature is graduated from Alpha to Beta.
+
+### Added
+
+- Support for proxying all Service traffic by Antrea Proxy, including NodePort, LoadBalancer, and ClusterIP traffic. Therefore, running kube-proxy is no longer required. ([#2599](https://github.com/antrea-io/antrea/pull/2599) [#2235](https://github.com/antrea-io/antrea/pull/2235) [#2897](https://github.com/antrea-io/antrea/pull/2897) [#2863](https://github.com/antrea-io/antrea/pull/2863), [@hongliangl] [@lzhecheng])
+  * The feature works for both Linux and Windows
+  * The feature is experimental and therefore disabled by default. Use the `antreaProxy.proxyAll` configuration parameter for the Antrea Agent to enable it
+  * If kube-proxy is removed, the `kubeAPIServerOverride` configuration parameter for the Antrea Agent must be set to access kube-apiserver directly.
+- Add [AntreaIPAM] feature that allows flexible control over Pod IP Addressing by assigning pools of IP addresses to specific Namespaces. ([#2956](https://github.com/antrea-io/antrea/pull/2956), [@gran-vmv] [@annakhm])
+  * Add new IPPool API to define ranges of IP addresses which can be used as Pod IPs; the IPs in the IPPools must be in the same "underlay" subnet as the Node IP
+  * A Pod's IP will be allocated from the IPPool specified by the `ipam.antrea.io/ippools` annotation of the Pod's Namespace if there is one
+  * When the feature is enabled, the Node's network interface will be connected to the OVS bridge, in order to forward cross-Node traffic of AntreaIPAM Pods to the underlay network
+  * Refer to the [documentation](https://github.com/antrea-io/antrea/blob/v1.4.0/docs/feature-gates.md#nodeipam) for more information
```

Comment:
@jianjuns I have updated the description of AntreaIPAM accordingly. Please check if the list look good to you.

Reply:
documentation -> feature documentation?